src/Storages/MergeTree/AsyncBlockIDsCache.cpp

```diff
@@ -33,24 +34,7 @@ template <typename TStorage>
 std::vector<String> AsyncBlockIDsCache<TStorage>::getChildren()
 {
     auto zookeeper = storage.getZooKeeper();
```

Comment:
This function became very small and its name 'getChildren' doesn't make much sense in the context of block id cache. I suggest to just move these 2 line into  `AsyncBlockIDsCache<TStorage>::update()`